### PR TITLE
Added documentation for branching_weight() solving issue #5553

### PR DESCRIPTION
**Pull request by NikitaSharma1**
I solved #5553 issue and added parameters, returns, and examples in the branching_weight() function.

### CODE REVIEW

**Comment by dschult** [comment]
The documentation from the "show all checks"/"details"/"document artifact" link shows [what the new doc_string](https://output.circle-artifacts.com/output/job/6ead1d27-9815-439b-9123-34eaa52f40ad/artifacts/0/doc/build/html/reference/algorithms/tree.html?highlight=branching#module-networkx.algorithms.tree.branchings) appears like.  (The document artifact is usually near the bottom of the list of checks).

At the module level, the whole doc_string appears in an unformatted way as the "one line description" of the function.
You need a blank line after the first line of the docstring.

The [docstring for the actual function itself](https://output.circle-artifacts.com/output/job/6ead1d27-9815-439b-9123-34eaa52f40ad/artifacts/0/doc/build/html/reference/algorithms/generated/networkx.algorithms.tree.branchings.branching_weight.html#networkx.algorithms.tree.branchings.branching_weight) also doesn't get converted completely. More blank lines are needed between the sections of the docstring.  Perhaps mimic the blank lines in a function that does format correctly.

Thanks!!

**Comment by NikitaSharma1** [comment]
@dschult Sir, I have made the changes and added blank lines.

File: networkx/algorithms/tree/branchings.py [hunk 1]
```diff
@@ -76,6 +76,32 @@ def branching_weight(G, attr="weight", default=1):
...
+    -------
+    weight: int or float
+        The total weight of the branching.
+
+    Examples
+    --------
+    >>> from networkx.algorithms import tree
+    >>> G = nx.DiGraph()
+    >>> G.add_weighted_edges_from([(0, 1, 2), (1, 2, 4), (2, 3, 3), (3, 4, 2)])
+    >>> tree.branching_weight(G)
```
Comment (dschult) [inline networkx/algorithms/tree/branchings.py:102]: It's actually easier to get to that using an import.
`nx.tree.branching_weight(G)` should work. Sorry I didn't catch that before... :}

Comment (NikitaSharma1) [reply networkx/algorithms/tree/branchings.py]: No problem!! I updated it.

**Review by dschult — APPROVED** [review]
Looks great!
Thanks!

**Pull request merged**